build-tools/automation/azure-pipelines.yaml

```diff
@@ -0,0 +1,128 @@
+# Java.Interop Pipelines
+
+trigger:
+  - master
+  - d16-*
+
+# Global variables
+variables:
+  DotNetCoreVersion: 2.1.701
+  HostedMac: Hosted Mac Internal
+  HostedWinVS2019: Hosted Windows 2019 with VS2019
+
+stages:
+  # Build and test on Windows
+- stage: windows_build
+  displayName: Windows
+  pool: $(HostedWinVS2019)
+  jobs:
+  - job: windows_build
```

Comment:
If you want to retain the Debug and Release configuration matrix, the following syntax can be used:

```yaml
- job: windows_build
  strategy:
    matrix:
      Debug:
        Build.Configuration: Debug
      Release:
        Build.Configuration: Release
```
This should automatically create two jobs and run the same steps for each variable value. Since `Build.Configuration` is already in use in later steps in the pipeline, I _think_ this should be a seamless change.

Reply:
Thanks for the tip!  I'm not sure there's much value in testing both configurations.  I did it previously because I didn't know what to do.  Now that it is driven by the `$(Build.Configuration)` variable we can easily kick off a `Debug` build if we ever wanted to for some reason.

Reply:
Works for me 👍